runtime/doc/lsp.txt

```diff
@@ -1316,6 +1316,13 @@ incoming_calls()                                *vim.lsp.buf.incoming_calls()*
     window. If the symbol can resolve to multiple items, the user can pick one
     in the |inputlist()|.
 
+inlay_hint({enable}, {bufnr})                       *vim.lsp.buf.inlay_hint()*
+    Enable/disable/toggle inlay hints for a buffer
+
+    Parameters: ~
+      • {enable}  (boolean|nil) true/false to enable/disable, nil to toggle
+      • {bufnr}   (integer|nil) Buffer handle, or nil for current
```

Comment:
Convention is:

- 0 = current buffer
- `nil` = all buffers (don't necessarily need to implement this here)

Reply:
done

Reply:
Would be nice to do this in a follow-up, though, as now you can't just map to `vim.lsp.buf.inlay_hints()` (like you can with most other `vim.lsp.buf` functions).